### PR TITLE
47701 : Memorise password

### PR DESCRIPTION
**Pull request by wbenabdessalem**
Given I am a mobile user
When I go to the authentification page
And I enter my username and password, I check the checkbox "stay connected" Or I connect using one of the available options (Google, linkedin..)
Then I am successfully logged to the application
------
Given the previous step
When I close the application and reopen it after a few time
Then I am still connected to the application and I don't have to login again

### CODE REVIEW

File: app/src/main/java/org/exoplatform/activity/BoardingActivity.java [hunk 1]
```diff
@@ -203,35 +211,86 @@ public void statusBarColor(){
...
         this.startActivity(intent);
     }
+
+    private void setWakeUpActivityRoot(final ResultHandler<Boolean> handler) throws IOException {
+        SharedPreferences prefs = App.Preferences.get(this);
+        Server serverToConnect = new ServerManagerImpl(prefs).getLastVisitedServer();
+        String username = prefs.getString("connectedUsername","username");
+        String cookies = prefs.getString("connectedCookies","cookies");
+        if (serverToConnect != null) {
+            final String url = serverToConnect.getUrl().getProtocol() + "://" + serverToConnect.getShortUrl() + "/portal/rest/state/status/" + username;
```
Comment (ahamdi) [inline app/src/main/java/org/exoplatform/activity/BoardingActivity.java]: COuld you put this in a global variable to simplify changing it if needed

File: app/src/main/java/org/exoplatform/activity/BoardingActivity.java [hunk 2]
```diff
@@ -203,35 +211,86 @@ public void statusBarColor(){
...
+            System.out.println("url =========> " + url);
```
Comment (ahamdi) [inline app/src/main/java/org/exoplatform/activity/BoardingActivity.java]: Let's use Logger instead of printing directly on stdout

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/exoplatform/activity/BoardingActivity.java (hunk 2 above)
Comment (wbenabdessalem) [reply app/src/main/java/org/exoplatform/activity/BoardingActivity.java]: Done

File: app/src/main/java/org/exoplatform/activity/BoardingActivity.java (hunk 1 above)
Comment (wbenabdessalem) [reply app/src/main/java/org/exoplatform/activity/BoardingActivity.java]: Done

**Review by ahamdi — APPROVED** [review]

**Pull request merged**